src/cloud/integrations/gitlab-integration.md

```diff
@@ -0,0 +1,161 @@
+
+   ```terminal
+   Hook URL: https://eu-3.magento.cloud/api/projects/3txxjf32gtryos/integrations/eolmpfizzg9lu/hook
+   Created integration eolmpfizzg9lu (type: gitlab)
+   +----------------------------------+---------------------------------------------------------------------------------------+
+   | Property                         | Value                                                                                 |
+   +----------------------------------+---------------------------------------------------------------------------------------+
+   | id                               | eolmpfizzg9lu                                                                         |
+   | type                             | gitlab                                                                                |
+   | token                            | ******                                                                                |
+   | base_url                         | https://gitlab.com/                                                                   |
+   | project                          | my-agency/project-name                                                            |
+   | fetch_branches                   | true                                                                                  |
+   | prune_branches                   | true                                                                                 |
+   | build_merge_requests             | false                                                                                  |
+   | merge_requests_clone_parent_data | false                                                                                  |
+   | hook_url                         | https://eu-3.magento.cloud/api/projects/3txxjf32gtryos/integrations/eolmpfizzg9lu/hook |
+   +----------------------------------+---------------------------------------------------------------------------------------+
+   ```
+   {:.no-copy}
+
+## Add the webhook in GitLab
+
+In order to communicate events —such as a push or merge requests— with your Cloud Git server, you need to [create a webhook](https://docs.gitlab.com/ee/user/project/integrations/webhooks.html#overview) for your GitLab repository
+
+1. In your GitLab repository, click the **Settings** tab.
+
+1. In the left navigation bar, click **Webhooks**.
+
+1. In the _Webhooks_ form, edit the following fields:
+
+   -  **URL**: Enter the `Hook URL` returned when you enabled the GitLab integration.
+   -  **Secret Token**: Enter a verification secret if needed.
+   -  **Trigger**: Check `Merge request events` and/or `Push events` depending on your needs
+   -  **Enable SSL verification**: Check it !
+
+1. Click **Add webhook**.
+
+## Test the integration
+
+To verify the integration works, make a change in the GitLab repository and use the magento-cloud CLI to pull the change into the local environment or just merge a request in GitLab.
```

Comment:
The test the integration information is not detailed enough.  In the Bitbucket topic, the instructions are clearer.

Reply:
Thanks for your review ;)
I added link to bitbucket test chapter instead of ust duplicate it and add a screen of a successful result.
Tell me if it's OK for you